ajax/libs/kurento-client/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "kurento-client",
+  "npmName": "kurento-client",
+  "npmFileMap": [
+    {
+      "basePath": "/src/main/resources/META-INF/resources/js/",
```

Comment:
Let's remove the leading and trailing slash of basePath like `"src/main/resources/META-INF/resources/js"`.